lib/grammars.coffee

```diff
@@ -273,6 +273,17 @@ module.exports =
       command: "racket"
       args: (context) -> [context.filepath]
 
+  RANT:
+    "Selection Based":
+      command: "RantConsole.exe"
```

Comment:
The language is for MS .Net Framework on Windows, but I have it running on a Mac under [Mono](http://www.mono-project.com/). However, my Atom-fu is white-belt level and I'm not clear on how to change the `command` and `args` based on the host platform (Win/Mac/Linux). 
On Windows you'd just run `RantConsole.exe -file FILE`, while on Mac/Linux you'd run `/usr/bin/mono /path_to_RANT/RantConsole.exe -file FILE`. I replicated the Windows method with a shell script on my dev machine; I figured folks who got RANT running on Mac/Linux would be likely to think of the same.